bdrcmodels/models.py

```diff
@@ -412,6 +412,8 @@ class TeiFile(CommonMetadataDO):
 class TxtFile(CommonMetadataDO):
     CONTENT_MODELS = [TXT_CONTENT_MODEL, COMMON_METADATA_CONTENT_MODEL]
 
+    content_ds_name = 'txt'
```

Comment:
If we go this way, wouldn't we use the same case as the FileDatastream?
In this case "TXT"